src/next-transpile-modules.js

```diff
@@ -187,6 +188,30 @@ const withTmInitializer = (modules = [], options = {}) => {
           `**node_modules/{${modules.map((mod) => `!(${mod})`).join(',')}}/**/*`,
         ];
 
+        if (isWebpack5) {
+          const checkForTranspiledModules = (currentPath) =>
```

Comment:
we need to check if its symlinked or not before excluding it from webpack cache, otherwise we are rebuilding uncached modules that are simply in ES6, but not in the monorepo as a workspace package